src/p_mobj.cpp

```diff
@@ -3163,6 +3167,53 @@ void AActor::SetRoll(angle_t r, bool interpolate)
 	}
 }
 
+void AActor::SetThrustFactor(FName dmgtype, fixed_t amount)
+{
+	if (!dmgtype || dmgtype == NAME_None || !stricmp("none", dmgtype) || !stricmp("normal", dmgtype))

```

Comment:
That's just it. I was ensuring case insensitivity was used.

But okay I can change those last two down to a simple 'dmgtype == NAME_Normal'